test/clientUnit.spec.ts

```diff
@@ -461,7 +483,6 @@ describe('BitPaySDK.Client', () => {
       invoice.notificationEmail = 'notification@email.com';
       invoice.autoRedirect = true;
       invoice.forcedBuyerSelectedWallet = 'bitpay';
-      invoice.forcedBuyerSelectedTransactionCurrency = null;
```

Comment:
Similar to other file -- `forcedBuyerSelectedTransactionCurrency` should be a valid request param.